ecs.tf

```diff
@@ -4,6 +4,7 @@ data "aws_ecs_cluster" "target_cluster" {
 
 resource "aws_ecs_task_definition" "main_task" {
   family                = "${var.app_name}-tsk"
+  requires_compatibilities = [var.launch_type]
   network_mode          = "bridge"
   cpu                   = var.cpu
   memory                = var.memory
```

Comment:
Could you keep the spacing consistent? Will need to add spaces to the other args.